product_docs/docs/edbcloud/beta/using_cluster/02_connect_to_cluster.mdx

```diff
@@ -12,4 +14,14 @@ To connect to your cluster using `psql`:
 To connect to your cluster using a client other than `psql`:
 1. Go to the [**Clusters**](https://portal.edbcloud.com/clusters) page.
 2. Select the name of your cluster. 
-3. Select the **Connect** tab. You can review and copy all the relevant information you need from this screen except for the `edb_admin` user password. Please consult the client driver documentation for the connection string format the driver uses.  
+3. Select the **Connect** tab. You can review and copy all the relevant information you need from this screen except for the `edb_admin` user password. Please consult the client driver documentation for the connection string format the driver uses.
+
+## Best Practice Recommendation - SSL Mode
+
+Different clients can have different default SSL modes. For example, `psql` defaults to `prefer`, which means the client will attempt to establish a SSL connection but fall back to non-SSL if the server does not support it. In the `psql` example provided by EDB in the **Quick Connect** field, SSL mode is explicitly set to `require`, which means the client will attempt an SSL connection and fail if the connection to the server can't be encrypted. 
```

Comment:
Can we do some sort of wordsmithing to note that we don't actually support SSL, that we support TLS (v1.2+), but PG still talks about SSL for historical reasons?

Reply:
@ba-edb Let me know if you agree with the latest changes.